codeql-custom-queries-cpp/qlpack.yml

```diff
@@ -1,3 +1,4 @@
-name: codeql-custom-queries-cpp
+name: codeql/custom-queries-cpp
```

Comment:
These are often used as a template when creating one's own query packs. Would it be better to use a different scope name?

Reply:
I'll change this and also add a comment for users to use a scope they have access to.